### PR TITLE
Added how to install via composer

### PR DESCRIPTION
**Pull request by royopa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
 
 Buzz is a lightweight PHP 5.3 library for issuing HTTP requests.
 
+## Instalation
+
+Package available on [Composer](https://packagist.org/packages/kriswallsmith/buzz).
+
+If you're using Composer to manage dependencies, you can use
+
+    $ composer require "kriswallsmith/buzz": "dev-master"
```
Comment (ged15) [inline README.md]: why not just `composer require "kriswallsmith/buzz"`? Composer resolves the best version automatically.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,28 @@
...
+
+Or
```
Comment (ged15) [inline README.md]: http://blog.doh.ms/2014/10/13/installing-composer-packages/ :)

**Comment by royopa** [comment]
Hi @ged15! I updated the readme.
Regards!

File: README.md [hunk 3]
```diff
@@ -2,6 +2,28 @@
...
+    $ composer require "kriswallsmith/buzz"
+
+Or
+
+You can include the following in your composer.json file:
+
+```json
+{
+    "require": {
+        "kriswallsmith/buzz": "*"
```
Comment (stof) [inline README.md]: please don't advocate the usage of the ``*`` constraint (I suggest keeping the ``composer require`` way only)

Comment (ged15) [inline README.md]: ditto

**Comment by royopa** [comment]
Ok, I updated the readme

**Comment by Nyholm** [comment]
Thank you for this PR.

**Pull request merged**